### DISCUSSION TITLE
Run last day of month

### DISCUSSION THREAD

**Discussion by SveinnB** [jamesmh/coravel#401] [Q&A] [answered]
Hi

What is the best/recomended way of create a Task Scheduling that run last day of every month?

**Comment by jamesmh** [comment, marked as answer]
There are some options as mentioned in here https://github.com/jamesmh/coravel/issues/382

There's no built in method to do this, specifically. But you can use the `When()` method to do this on your own by checking if the next day runs into the next month:

```csharp
scheduler.Schedule<Job>()
   .DailyAtHour(12)
   .When(() =>
   {
      var now = DateTime.UtcNow;
      return now.Month != now.Add(TimeSpan.FromDays(1)).Month;
   });
```
